sysutils/qmk/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                qmk
+version             1.1.1
+revision            0
+
+categories-prepend  sysutils
+
+license             MIT
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description         CLI tool for customizing supported mechanical keyboards.
+long_description    {*}${description}
+
+homepage            https://github.com/qmk/qmk_cli
+
+checksums           rmd160  196cb96b9cdc85b705eb0115aa034d9da857f6bb \
+                    sha256  dd028e09ebcd61f8bdf8cb82929dfafc0e007d97a5a3803b45819b4641773269 \
+                    size    11994
+
+python.default_version \
+    310
```

Comment:
Python 3.21 is the current default in MacPorts. The port should be updated to use this if possible.

Reply:
change to 3.11